Readme.md

```diff
@@ -1,39 +1,66 @@
-<!-- default file list end -->
-# ASPxGridView- How to conditionally enable and disable the batch editing on the client side
+# Grid View for ASP.NET Web Forms - How to control data editing on the client side in batch edit mode
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/t150957/)**
 <!-- run online end -->
 
+This example demonstrates how to use the check box state to control data editing in the grid in batch edit mode.
+
+![Allow Edit Checkbox](controlEditing.gif)
+
+## Overview
+
+Follow the steps below:
+
+1. Create the [Grid View](https://docs.devexpress.devx/AspNet/DevExpress.Web.ASPxGridView) control and populate it with columns. Set the grid's [SettingsEditing.Mode](https://docs.devexpress.devx/AspNet/DevExpress.Web.ASPxGridViewEditingSettings.Mode) property to `Batch` to enable the batch edit mode. Add a [GridViewCommandColumn](https://docs.devexpress.devx/AspNet/DevExpress.Web.GridViewCommandColumn) and set its [ShowNewButtonInHeader](https://docs.devexpress.devx/AspNet/DevExpress.Web.GridViewCommandColumn.ShowNewButtonInHeader) and [ShowDeleteButton](https://docs.devexpress.devx/AspNet/DevExpress.Web.GridViewCommandColumn.ShowDeleteButton) properties to `true`.
+
+    ```aspx
+    <dx:ASPxGridView ID="ASPxGridView1" runat="server" KeyFieldName="ID" ...>
+        <SettingsEditing Mode="Batch" />
+        <Columns>
+            <dx:GridViewCommandColumn ShowNewButtonInHeader="true" ShowDeleteButton="true" />
+            <!-- ... -->
+        </Columns>
+    </dx:ASPxGridView>
+    ```
 
-<p>This example demonstrates how to conditionally enable or disable the editing on the client side.</p>
-<p><strong>See also:</strong><br><a href="https://www.devexpress.com/Support/Center/p/T150965">GridView - Batch Editing - How to conditionally enable and disable the editing on the client side</a></p>
+2. Add a check box and handle its client-side [CheckedChanged](https://docs.devexpress.devx/AspNet/DevExpress.Web.ASPxCheckBox.CheckedChanged) event. In the handler, create a flag variable, get the current state of the check box, and assign this value to the flag variable.
 
+    ```aspx
+    <dx:ASPxCheckBox ID="ASPxCheckBox1" runat="server" Text="Allow edit">
+        <ClientSideEvents CheckedChanged="OnAllowEditChanged" />
+    </dx:ASPxCheckBox>
+    ```
 
-<h3>Description</h3>
+    ```js
+    var allowEdit = false;
+    function OnAllowEditChanged(s, e) {
+        allowEdit = s.GetValue();
+    }
+    ```
+
+3. Handle the grid's client-side [BatchEditStartEditing](https://docs.devexpress.devx/AspNet/js-ASPxClientGridView.BatchEditStartEditing), [BatchEditRowInserting](https://docs.devexpress.devx/AspNet/js-ASPxClientGridView.BatchEditRowInserting), and [BathEditRowDeleting](https://docs.devexpress.devx/AspNet/js-ASPxClientGridView.BatchEditRowDeleting) events. In the handler, cancel the current editing operation based on the flag variable value.
```

Comment:
> [BathEditRowDeleting](https://docs.devexpress.devx/AspNet/js-ASPxClientGridView.BatchEditRowDeleting) events.

"Bath" instead of "Batch". Another typo that is easily missed by any spellchecker. Please be more attentive.

> cancel the current editing operation

Is there any specific reason to use the gerund form? The word "edit" works well enough as an adjective. The current edit operation.

```suggestion
3. Handle the grid's client-side [BatchEditStartEditing](https://docs.devexpress.devx/AspNet/js-ASPxClientGridView.BatchEditStartEditing), [BatchEditRowInserting](https://docs.devexpress.devx/AspNet/js-ASPxClientGridView.BatchEditRowInserting), and [BatchEditRowDeleting](https://docs.devexpress.devx/AspNet/js-ASPxClientGridView.BatchEditRowDeleting) events. In the handler, cancel the current edit operation based on the flag variable value.
```